lib/resources/storage/web_storage.dart

```diff
@@ -10,4 +14,28 @@ class StorageProvider {
       return null;
     }
   }
+
+  static Future<bool> uploadImage(
+      BuildContext context, Uint8List file, String ref) async {
+    try {
+      final StorageReference storageReference = storage().ref('').child(ref);
+
+      final UploadTaskSnapshot uploadTaskSnapshot =
+          await storageReference.put(file).future;
+
+      if (uploadTaskSnapshot.state == TaskState.SUCCESS) {
+        return true;
+      } else {
+        return false;
```

Comment:
Same as above. log the errors and use the `onComplete` callback.

Reply:
Same as above